pupil_src/shared_modules/video_overlay/utils/image_manipulation.py

```diff
@@ -112,7 +118,8 @@ def render_ellipse(self, image, ellipse, color):
         outline = [np.asarray(outline, dtype="i")]
         cv2.polylines(image, outline, True, color, thickness=1)
 
-        center = (int(ellipse["center"][0]), int(ellipse["center"][1]))
+        center = ellipse["center"]
+        center = (float_to_int(center[0]), float_to_int(center[1]))
```

Comment:
Converting all fields from NaN to 0 was not my intention. Rather, I would prefer a behavior where the whole datum to be treated as if the _confidence_ was 0 if any of the relevant fields (confidence, ellipse center) is NaN-valued.